checkov/openapi/runner.py

```diff
@@ -46,7 +46,14 @@ def parse_format(
         func: _ParseFormatJsonCallable | _ParseFormatYamlCallable,
     ) -> tuple[dict[str, Any] | list[dict[str, Any]], list[tuple[int, str]]] | None:
         try:
-            parsed_file = func(self, f)
+            if f.endswith(".json"):
+                parsed_file = func(self, f)
```

Comment:
why not also do it for JSON files?

Reply:
1. The main pain for openapi resides in the YAMLs, I want to check first if it improves performance and doesn't raise any other issues.
2. YAML runner allows passing the file content to the parser, so we don't need to open and load the file twice. JSON runner currently does not allow passing the content, and modifying it requires a deeper and riskier change.

Reply:
sounds good 🙂